### PR TITLE
add support for paypal credit

### PR DESCRIPTION
**Pull request by slogsdon**
Reverts hps/heartland-woocommerce-plugin#102

### CODE REVIEW

File: classes/class-wc-gateway-securesubmit-paypal.php [hunk 1]
```diff
@@ -118,7 +146,9 @@ public function startIncontext()
             $this->setSession('checkout_form', $_POST);
         }
 
-        print json_encode($this->createSession->call(null));
+        $credit = isset($_POST['paypalexpress_credit']) || isset($_GET['paypalexpress_credit']);
```
Comment (ttnnkkrr) [inline classes/class-wc-gateway-securesubmit-paypal.php]: @slogsdon So literally any value is acceptable and can either be get or post? is the method really unknown?

File: classes/class-wc-gateway-securesubmit-paypal.php [hunk 2]
```diff
@@ -194,8 +224,9 @@ public function startExpressCheckout()
         $woocommerce = WC();
         $this->setSession('ss-paypal-express-checkout-inprogress', true);
         $this->setSession('checkout_form', $_POST);
+        $credit = isset($_POST['paypalexpress_credit']) || isset($_GET['paypalexpress_credit']);
```
Comment (ttnnkkrr) [inline classes/class-wc-gateway-securesubmit-paypal.php]: @slogsdon So literally any value is acceptable and can either be get or post? is the method really unknown?

File: classes/class-wc-gateway-securesubmit-paypal.php (hunk 1 above)
Comment (slogsdon) [inline classes/class-wc-gateway-securesubmit-paypal.php]: Using both `$_GET` and `$_POST` is to allow for options to allow for how the initial request is being made. The index won't be set if credit shouldn't be flagged.

File: classes/class-wc-gateway-securesubmit-paypal.php (hunk 2 above)
Comment (slogsdon) [inline classes/class-wc-gateway-securesubmit-paypal.php]: Using both `$_GET` and `$_POST` is to allow for options to allow for how the initial request is being made. The index won't be set if credit shouldn't be flagged.

**Comment by slogsdon** [comment]
I need to verify one thing. Will circle back with a commit or a comment soon.

**Pull request merged**